core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -358,9 +361,11 @@ class SparkContext(config: SparkConf) extends Logging {
     }
   }
 
-  /** Control our logLevel. This overrides any user-defined log settings.
+  /**
+   * Control our logLevel. This overrides any user-defined log settings.
+   *
    * @param logLevel The desired log level as a string.
-   * Valid log levels include: ALL, DEBUG, ERROR, FATAL, INFO, OFF, TRACE, WARN
+   *                 Valid log levels include: ALL, DEBUG, ERROR, FATAL, INFO, OFF, TRACE, WARN
```

Comment:
Yes we don't need continuation indents. The code doesn't use it in general and wastes some space